testsuite/features/step_definitions/api_common.rb

```diff
@@ -500,13 +500,13 @@ def wait_action_complete(actionid, timeout: DEFAULT_TIMEOUT)
   step %(I should get status "#{status}" for system "#{get_system_id(node)}")
 end
 
-Then(/^I should get the test channel$/) do
-  arch = `uname -m`
+Then(/^I should get the test base channel$/) do
+  arch, _code = get_target('server').run('uname -m')
   arch.chomp!
   channel = if arch != 'x86_64'
               'fake-base-channel-i586'
             else
```

Comment:
We already have servers running on s390, arm, and powerpc, and we will have ARM... and then we would choose i586 as base channel ? Does not make sense.

I would just nuke all that i586 hack...

Reply:
but anyway... not the purpose of that pr, as you said

approving

Reply:
We don't build, or we don't at least mirror, any test repositories other than those coming from these two channels.